src/utils/specs.py

```diff
@@ -310,14 +310,10 @@ def get_machine_specs(devicenum):
 
     rocm_version = rocm_ver.strip()
 
-    freq = search(device, rocm_smi).split()
-    cur_sclk = search(r"([0-9]+)", freq[2])
-    if cur_sclk is None:
-        cur_sclk = ""
-
-    cur_mclk = search(r"([0-9]+)", freq[3])
-    if cur_mclk is None:
-        cur_mclk = 0
+    # these are just max's now, because the parsing was broken and this was inconsistent
+    # with how we use the clocks elsewhere (all max, all the time)
+    cur_sclk = gpu_info['max_sclk']
+    cur_mclk = gpu_info['max_mclk']
```

Comment:
Currently, `max_mclk` isn't a field declared in gpu_info. This will casue:

```
Exception has occurred: KeyError
'max_mclk'
  File "/home/colramos/GitHub/omniperf-pub/src/utils/specs.py", line 317, in get_machine_specs
    cur_mclk = gpu_info['max_mclk']
  File "/home/colramos/GitHub/omniperf-pub/src/omniperf_base.py", line 153, in detect_soc
    mspec = get_machine_specs(0)
  File "/home/colramos/GitHub/omniperf-pub/src/utils/utils.py", line 45, in wrap_function
    result = function(*args, **kwargs)
  File "/home/colramos/GitHub/omniperf-pub/src/omniperf_base.py", line 201, in run_profiler
    targ_arch = self.detect_soc()
  File "/home/colramos/GitHub/omniperf-pub/src/utils/utils.py", line 45, in wrap_function
    result = function(*args, **kwargs)
  File "/home/colramos/GitHub/omniperf-pub/src/omniperf", line 39, in main
    omniperf.run_profiler()
  File "/home/colramos/GitHub/omniperf-pub/src/omniperf", line 50, in <module>
    main()
KeyError: 'max_mclk'
```

Reply:
Oh, d'oh: I had this defined in the HBM PR: https://github.com/AMDResearch/omniperf/pull/243/files, but not here.
So we should probably resolve^ first, and this will work.